### PR TITLE
Ftr ui integration

### PR DESCRIPTION
**Pull request by ftreguer**

### CODE REVIEW

**Comment by dlecan** [comment]
Deploy preview for *www-sonarplanet* processing.

Built with commit 0928c78e73b58085f036b989420d6306963be5bd

https://app.netlify.com/sites/www-sonarplanet/deploys/5b0e5b2ab13fb16bf8dcdf6b

**Comment by Nigui** [comment]
Very good job.

File: src/styles/main.scss [hunk 1]
```diff
@@ -78,97 +157,464 @@
...
+      }
+    }
+    .universal {
+      background: linear-gradient(180deg, #FFFFFF 0%, #d7d7d7 100%);
+    }
+    .no-signup {
+      background: linear-gradient(180deg, $color-c3 0%, $color-c2 100%);
+      color: $color-c5;
+      box-shadow: 0px 10px 60px 0px #000000;
+    }
```
Comment (Nigui) [inline src/styles/main.scss]: I would add an index on middle card in order to add shadow on both left and right cards.

`z-index:1`

**Review by Nigui — COMMENTED** [review]
Could be good to animate text in header with this effect : https://tympanus.net/Tutorials/CSS3RotatingWords/index.html

Watch "blockchain transaction" & get notified
Watch "transaction confirmations" & get notified
Watch "address balance" & get notified

**Review by Nigui — COMMENTED** [review]
When you click on arrow in "action" section and you are at the first step, it goes back to an empty view. Maybe we can disable back button during first step ?

When you click twice on back arrow from the first step, you go to the third step.

File: src/index.html [hunk 2]
```diff
@@ -44,335 +43,472 @@
...
-                                                <input type="text" class="form-control" id="balanceFloorValue"> #NETWORK_CRYPTO
+                                        <div class="col-md-4"></div>
+                                    </div>
+
+                                    <div class="form-group">
+                                        <div class="collapse" id="collapseConfirmations">
+                                            <div class="card card-body">
+                                                <div>
+                                                    <div>
+                                                        <label class="sonar-radio">Get notified for each confitmation until validation (7 confirmations)
```
Comment (Nigui) [inline src/index.html]: confitmation --> confirmation

File: src/index.html [hunk 3]
```diff
@@ -44,335 +43,472 @@
...
+                                    <div class="row">
+                                        <div class="col-md-4"></div>
+                                        <div class="form-group col-md-4">
+                                            <div class="sonar-checkbox">
+                                                <label class="switch">
+                                                    <input type="checkbox" id="incomingTransactionCbx" value="incoming-transaction" aria-label="INCOMING TRANSACTION">
+                                                    <span class="slider round"></span>
                                                 </label>
+                                                INCOMING TRANSACTION
+                                                <span class="fa-layers fa-fw" data-toggle="tooltip" data-placement="top" title="Tootltip informations">
```
Comment (Nigui) [inline src/index.html]: Do we show tooltip icons even if it does nothing on hover ?

**Review by Nigui — COMMENTED** [review]
The dropdown menu for network search is hidden by community-section.

Maybe add a `z-index` on one of those div

File: src/ts/main.ts [hunk 4]
```diff
@@ -20,12 +24,12 @@ $(document).ready(() => {
         startTextAnimation(0);
       }, 2000);
```
Comment (Nigui) [inline src/ts/main.ts]: no stop between loops if you set it to `0`.
Looks better from my point of view.

**Review by Nigui — COMMENTED** [review]
maybe we can increase writing speed, dividing value by two on this line : https://github.com/sonarplanet/www.sonarplanet.io/blob/0928c78e73b58085f036b989420d6306963be5bd/src/ts/main.ts#L13

What do you think ?

File: src/ts/main.ts (hunk 4 above)
Comment (ftreguer) [reply src/ts/main.ts]: Yep, much better! ;)

File: src/index.html [hunk 5]
```diff
@@ -44,335 +43,472 @@
...
-                            <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Odio iusto hic repellat consectetur
-                                error iure voluptate.</p>
+                    <div class="col-sm-6 col-md-4 simple">
+                        <div class="thumbnail">
+                            <img src="./images/lock.svg" alt="lock">
+                            <div class="caption">
+                                <h3>SIMPLE</h3>
+                                <p>
+                                    Sonar Planet is totally private, you choose your level of anonymity. No email and no phone required, unless you want to receive
+                                    your notification.
```
Comment (Nigui) [inline src/index.html]: I would change it by `unless you want to be notified via these channels`

File: src/index.html [hunk 6]
```diff
@@ -44,335 +43,472 @@
...
-                        <label>Email</label>
-                        <input name="entry.1636836867" type="email" required placeholder="email@domain.com" />
-                    </div>
-                    <div>
-                        <input type="submit" value="Send" />
+                    <div class="input-group my-4">
+                        <input id="newsletter-email" name="entry.1636836867" type="email" class="form-control" placeholder="Enter your email address"
+                            aria-label="Enter your email address" aria-describedby="basic-addon2">
+                        <div class="input-group-append">
+                            <input class="input-group-text" id="basic-addon2" type="submit" value="Subscribe" />
```
Comment (Nigui) [inline src/index.html]: it works great

File: src/index.html [hunk 7]
```diff
@@ -44,335 +43,472 @@
...
+                        </div>
                     </div>
+
                 </form>
             </div>
-            <div>
-                <span class="upper-text">(privacy is in our core value, we will never sell nor share your email)</span>
+            <div class="privacy">
+                (Privacy is in our core value, we will never sell nor share your email)
```
Comment (Nigui) [inline src/index.html]: `Privacy is in our core values`, `s` is missing ?

File: src/index.html [hunk 8]
```diff
@@ -44,335 +43,472 @@
...
+                    </h6>
+                    <div class="slack">
+                        <iframe name="hidden_iframe_slack" id="hidden_iframe_slack" style="display:none;"></iframe>
+                        <form id="slackForm" action="https://docs.google.com/forms/d/e/1FAIpQLScU4HMtz2nTZe8ffUXbBJF0SbWz-vUW4-_GpkCvYcsN4gQYeQ/formResponse"
+                            method="post" target="hidden_iframe_slack">
+                            <div class="input-group my-4">
+                                <input id="slack-join" name="entry.646647047" type="email" class="form-control" placeholder="Enter your email to get an invite"
+                                    aria-label="Enter your email to get an invite" aria-describedby="basic-addon2">
+                                <div class="input-group-append">
+                                    <input class="input-group-text" id="basic-addon2" type="submit" value="GO!" />
```
Comment (Nigui) [inline src/index.html]: Works great !

**Review by Nigui — COMMENTED** [review]
Responsivity is broken on smartphones 

On very large devices (>1600), `action` section's height is huge.

**Comment by Nigui** [comment]
`slackForm.ts` and `eMailForm.ts` could be merged into one single file

File: src/index.html (hunk 2 above)
Comment (ftreguer) [reply src/index.html]: Done!

File: src/index.html (hunk 3 above)
Comment (ftreguer) [reply src/index.html]: I'm waiting for tooltips

File: src/index.html (hunk 5 above)
Comment (ftreguer) [reply src/index.html]: Done!

**Comment by ftreguer** [comment]
Vertical slider done too!

**Review by Nigui — APPROVED** [review]
great

**Pull request merged**